tools/local/check-cluster-health.sh

```diff
@@ -11,8 +11,27 @@ upload_operational=$([ $? -eq 0 ] && echo true || echo false)
 CONTAINER_NAME=bfb-search tools/local/check-docker-exit-code.sh
 search_operational=$([ $? -eq 0 ] && echo true || echo false)
 
-tools/check-consistency.py
-is_data_consistent=$([ $? -eq 0 ] && echo true || echo false)
+is_data_consistent=true
+pids=()
+
+for i in {1..5}; do
+  (
+    tools/check-consistency.py
+    if [ $? -ne 0 ]; then
+      exit 1
+    fi
+  ) &
+  pids+=($!)
+done
+
+for pid in "${pids[@]}"; do
+  wait $pid
+  echo "level=\"Process finished\" pid=$pid"
+  if [ $? -ne 0 ]; then
+    is_data_consistent=false
+    break
+  fi
+done
```

Comment:
Though I'm fine with this, you can also just use `wait` once without PIDs to wait for all background jobs

Reply:
I agree. Done

Reply:
I thought it returned a non-zero exit code if any of the childs failed, but that doesn't appear to be the case.

Let's leave it like this :+1: